Rakefile

```diff
@@ -0,0 +1,21 @@
+$LOAD_PATH.unshift File.join(File.dirname(__FILE__), "lib")
+
+require 'opsgenie_alert_tagging'
+require 'opsgenie_alert'
+require 'active_support/time'
+
+require "rspec/core/rake_task"
+
+RSpec::Core::RakeTask.new(:spec)
+
+task default: %i[spec]
+
+namespace :opsgenie_alert_tagging do
```

Comment:
Do we need such a verbose namespace? I suggest `opsgenie_alerts` instead. That way you run `rake opsgenie_alerts:add_tags`. You could also go for `rake opsgenie:tag_alerts` for something even shorter.